core-profile/cdi-langmodel-tck/pom.xml

```diff
@@ -164,6 +164,9 @@
                             <classpathDependencyExcludes>
                                 <exclude>jakarta.enterprise:jakarta.enterprise.cdi-api</exclude>
                                 <exclude>jakarta.enterprise:jakarta.enterprise.lang-model</exclude>
+                                <exclude>jakarta.ejb</exclude>
+                                <exclude>jakarta.el</exclude>
+                                <exclude>org.glassfish</exclude>
```

Comment:
Per [classpathDependencyExclude](https://maven.apache.org/surefire/maven-surefire-plugin/test-mojo.html#classpathDependencyExcludes) and its referenced https://maven.apache.org/plugins/maven-assembly-plugin/advanced-descriptor-topics.html#advanced-artifact-matching-in-includes-and-excludes this isn't valid; it needs to at least be a G:A or include a wildcard.

This and the PR diff section above need comments as it's not at all intuitive. The  PR diff section above creates a list of jars, and then right before this classpathDependencyExcludes we add those as additionalClasspathElements. But then we remove analogous deps here.

Thinking about it I'm confident it's all about using the jars from our modules and not the related maven deps. That could use a comment.

Reply:
Hmm.... ....interesting that it seems to work, but I agree we should use wildcards. I'll also make comments on this as well.